### ISSUE TITLE
linuxboot is slower than uefi

### ISSUE THREAD

**Issue by hanxinwu1** [linuxboot/linuxboot#81]
When I run Linuxboot on a server, the server boot becomes slower, the linuxboot project has to load the Linux kernel twice, the device will be loaded twice, is there any way to reduce the boot time of Linuxboot?Thanks!

**Comment by rminnich** [comment]
how are you loading it? Are you booting LinuxBoot from UEFI?

**Comment by hanxinwu1** [comment]
> how are you loading it? Are you booting LinuxBoot from UEFI?
Yes, I replaced the uefi shell of the UEFI file with the Linux kernel, and embedded the u-root compiled initramfs file, loaded the operating system with the localboot -grub command, and found it to be slower.
